### PR TITLE
3 sum added in python3 and js

### PR DESCRIPTION
**Pull request by alexanderywang**
all comments/edits welcome.

### CODE REVIEW

File: coding/3sum/meta.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+identifier: 3Sum
+category: coding
+subcategory: "array"
+difficulty: medium
+description:
+  en: README_en.md
+solution:
+  en: SOLUTION_en.md
+starter_code:
+  c: "starter.c"
```
Comment (marshallshen) [inline coding/3sum/meta.yml:11]: if only js and python files are provided, you can remove other languages here:
```suggestion
starter_code:
  python: "starter.py"
  javascript: "starter.js"
```

**Review by marshallshen — COMMENTED** [review]
Thank you so much for contributing! I have one comment but otherwise it looks good!
Can you also rebase from main and repush your branch? I added some CI/CD checks for this repo 😄 

Also a relevant PR I just created: https://github.com/HackerPen/tech-interview-questions/pull/4

File: coding/3sum/meta.yml (hunk 1 above)
Comment (alexanderywang) [reply coding/3sum/meta.yml:11]: cool. Just so I understand, in the meta.yml, i should only include starter_code for languages that I provide? In future contributions, are you looking for starter_code for all languages?  Or just will you fill in the rest?  Thanks!

Comment (marshallshen) [reply coding/3sum/meta.yml:11]: You can fill in as many languages as you can, and the community can always add in more in the future.

**Comment by alexanderywang** [comment]
> Looks like your pull request includes `two_sum` related questions 🤔 See if you can resolve those, if not, it's okay to merge too :)

hmm. i seem to have those files after pulling while rebasing

**Pull request merged**